<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GestionConfig</class>
 <widget class="QDialog" name="GestionConfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>150</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="EditButton">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>20</y>
     <width>250</width>
     <height>110</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 15px;
border: 1px solid black;
background-color: rgb(230, 230, 230);</string>
   </property>
   <property name="text">
    <string>EDIT CONFIGURATION</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ReturnButton">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>20</y>
     <width>120</width>
     <height>110</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 15px;
border: 1px solid black;
background-color: rgb(230, 230, 230);</string>
   </property>
   <property name="text">
    <string>Return</string>
   </property>
  </widget>
  <widget class="QPushButton" name="NewButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>250</width>
     <height>110</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 15px;
border: 1px solid black;
background-color: rgb(230, 230, 230);</string>
   </property>
   <property name="text">
    <string>NEW CONFIGURATION</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
